lib/irb.rb

```diff
@@ -463,6 +463,10 @@ class Irb
     # be parsed as :assign and echo will be suppressed, but the latter is
     # parsed as a :method_add_arg and the output won't be suppressed
 
+    PROMPT_MAIN_TRUNCATE_LENGTH = 32
```

Comment:
Why did you choose 32?
(No intention to disagree)

Reply:
No particular reason. I thought 30-40 would be appropriate, just choose 32.

`Object.new.to_s.size => 28` so it should be >= 28